enhancements/sig-architecture/82-addon-template/examples/addon-template.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: AddOnTemplate
+metadata:
+  name: hello-template
+spec:
+  addonName: hello-template
+  agentManifests:
```

Comment:
Is the primary benefit of these `agentManifests` to provide a way to have the spokes start running a pod without the add-on  having to provide a hub-side controller to create manifest-works or other distribution method?

Reply:
it is still to use manifestwork to distribute, but it is to avoid each add-to providing a hub-side controller.